pages/linux/grub-script-check.md

```diff
@@ -0,0 +1,21 @@
+# grub-script-check
+
+> The program grub-script-check takes a GRUB script file and checks it for syntax errors.
```

Comment:
```suggestion
> The program `grub-script-check` takes a GRUB script file and checks it for syntax errors.
```

Reply:
Nice suggestion. GitHub renders backticks in a more cute manner than plain double quotes. :)